Tensile/Source/client/include/HardwareMonitor.hpp

```diff
@@ -143,6 +152,9 @@ namespace Tensile
 
             std::vector<uint32_t> m_fanMetrics;
             std::vector<int64_t>  m_fanValues;
+            std::vector<uint16_t> m_freqValues;
+            std::vector<uint16_t> m_powerValues;
+            std::vector<uint16_t> m_temperatureValues;            
```

Comment:
Can we call this something more consistent like m_tempHotspotValues? We already have a m_tempValues and this makes it confusing.

Reply:
>  Need to either fix it to be consistent with the other metrics or handle the edge case
Yes working on handling the edge case.

>Can we call this something more consistent like m_tempHotspotValues? 
yes, you are correct.